### PR TITLE
cmd/kubeadm/:add test file for data.go

### PR DESCRIPTION
**Pull request by Octopusjust**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Some test files are missing
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: cmd/kubeadm/app/cmd/phases/reset/data_test.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2019 The Kubernetes Authors.
```
Comment (neolit123) [inline cmd/kubeadm/app/cmd/phases/reset/data_test.go:2]: 2022

File: cmd/kubeadm/app/cmd/phases/reset/data_test.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+	"io"
+
+	"k8s.io/apimachinery/pkg/util/sets"
+	clientset "k8s.io/client-go/kubernetes"
+
+	kubeadmapi "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
+)
+
+// a package local type for testing purposes.
+type testresetData struct{}
```
Comment (neolit123) [inline cmd/kubeadm/app/cmd/phases/reset/data_test.go:29]: testData

File: cmd/kubeadm/app/cmd/phases/upgrade/node/data_test.go [hunk 3]
```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2019 The Kubernetes Authors.
```
Comment (neolit123) [inline cmd/kubeadm/app/cmd/phases/upgrade/node/data_test.go:2]: 2022

**Review by neolit123 — COMMENTED** [review]
ideally we want the naming consistent with other data_test files

**Comment by Octopusjust** [comment]
> ideally we want the naming consistent with other data_test files

I will modify these , thanks

**Comment by Octopusjust** [comment]
> /label tide/merge-method-squash You'd better squash the commits into one.
> 
> /ok-to-test

Got it , thanks

**Review by neolit123 — COMMENTED** [review]
thanks
/lgtm
/approve

**Pull request merged**